test/Middleware.js

```diff
@@ -0,0 +1,24 @@
+import {compile, serialize, stringify, middleware, rulesheet, prefixer, namespace} from "../index.js"
+
+const stack = []
+
+describe('Middleware', () => {
+	test('rulesheet', () => {
+  	serialize(compile(`.user{h1 {width:0;} @media{width:1;}}`), middleware([stringify, rulesheet(value => stack.push(value))]))
```

Comment:
the API looks slick 👍

Reply:
💯 It's 2X faster than current stylis(master branch)